templates/components/news-list/news-item-list.twig

```diff
@@ -26,49 +26,47 @@
 
 {% set news_url = news_url|render|striptags("<drupal-render-placeholder>")|trim %}
 
-<ul{{ attributes }}>
-  {# News List Date #}
-  {%- block block_news_list_date %}
-    {% if news_list_publishing_date|render|striptags('<drupal-render-placeholder>') is not empty %}
-      <li class="su-news-list__item-date">{{ news_list_publishing_date }}</li>
-    {% endif -%}
-  {% endblock -%}
-
+<article{{ attributes }}>
   {# News List Headline & Media #}
   {%- block block_news_list_headline  %}
-    <li class="su-news-list__item">
-      {%- if news_url is not empty -%}
-        <a {{ news_url_attributes }} href="{{ news_url }}">
-      {%- endif -%}
-      {% if news_list_image|render|striptags('<drupal-render-placeholder><img><picture>') is not empty -%}
+    <div class="news-list-header">
+    {%- if news_url is not empty -%}
+      <a {{ news_url_attributes }} href="{{ news_url }}">
```

Comment:
Hows the accessibility of this link?

Reply:
@cjwest had wanted us to move to only one link because two links to the same place is a validation error. We've been collaborating on this and making sure accessible. @cjwest Can you comment when you've pulled and tried it?

Reply:
@jenbreese Yes, we get an accessibility error if we have two links going to the same place.